pkg/resources/resources_support.go

```diff
@@ -146,3 +148,11 @@ func Resources(dirName string) ([]string, error) {
 
 	return res, dir.Close()
 }
+
+func fixPath(path string) string {
```

Comment:
Instead of creatng a custom function, it's better to use the builtin function `filepath.FromSlash()`
https://pkg.go.dev/path/filepath#FromSlash

Reply:
And since `assets.Open(filepath.ToSlash(path))` needs to be invoked every time as a set, why don't we insert the path fixing code inside the `Open` method directly here instead?
https://github.com/apache/camel-k/blob/main/pkg/resources/resources.go#L717